.github/workflows/ci.yml

```diff
@@ -17,60 +17,60 @@
 #
 
 ---
-name: Airflow
+name: Airflow CI Build
 on:
+  schedule:
+    # Schedule cron jobs every day at 2.10 am
+    - cron: '10 2 * * *'
   push:
-    branches: ['*']
+    branches: ['master', 'v1-10-test', 'v1-10-stable']
   pull_request:
-    branches: [master]
+    branches: ['master', 'v1-10-test', 'v1-10-stable']
 env:
-  BUILD_ID: ${{github.sha }}
   MOUNT_LOCAL_SOURCES: "false"
   MOUNT_HOST_AIRFLOW_VOLUME: "true"
   FORCE_ANSWER_TO_QUESTIONS: "yes"
   SKIP_CHECK_REMOTE_IMAGE: "true"
   DB_RESET: "true"
   VERBOSE: "true"
-  CI: "true"
```

Comment:
Why did re remove this? It seems like a useful thing to know/have.

Reply:
And is still used in scripts.

Reply:
CI is set to "true" automatically by both Travis and Github Actions.

- https://help.github.com/en/actions/configuring-and-managing-workflows/using-environment-variables
- https://docs.travis-ci.com/user/environment-variables/#default-environment-variables